### ISSUE TITLE
SMS code not autofilled

### ISSUE THREAD

**Issue by agungjk** [gurisko/cordova-plugin-accountkit#23]
Sms already received but it not autofilled to the form.

**Comment by gurisko** [comment]
Which OS?

**Comment by agungjk** [comment]
i'm using
Android 4.4.4 on XIAOMI device

Callback also not working if open accountkit ui thread opened

**Comment by gurisko** [comment]
Hm. Do you have the right permissions?
It should be
`<uses-permission android:name="android.permission.RECEIVE_SMS" />`

**Comment by agungjk** [comment]
@gurisko yes i did, but now the main issue is callback is not working if accountkit ui opened

**Comment by gurisko** [comment]
I tried to reproduce this but no success. Would you mind creating a repo with the code where this occurs?
